themes/vocabulary_theme/templates/blocks/featured-projects.html

```diff
@@ -1,40 +1,46 @@
-<section class="featured-projects">
+<section class="featured-projects" aria-labelledby="featured-projects-title">
```

Comment:
I'm not sure this is necessary since the child `div` elements will be ignored semanticially and a section's `h2` would automatically be used as a landmark to describe what it is?